pkg/domain/infra/abi/artifact.go

```diff
@@ -124,6 +125,14 @@ func (ir *ImageEngine) ArtifactRm(ctx context.Context, opts entities.ArtifactRem
 	for _, namesOrDigest := range namesOrDigests {
 		artifactDigest, err := artStore.Remove(ctx, namesOrDigest)
 		if err != nil {
+			if opts.Ignore {
+				logrus.Debugf(
+					"Artifact with name or digest: %s does not exist, ignoring error (--ignore): %v",
```

Comment:
I don't like to refer to cli options in output that far down in the code. The code could be called via other means such as the API in theory so let's avoid doing this.

How about:
```
logrus.Debugf("Artifact with name or digest %q does not exist, ignoring error as request", namesOrDigest)
```
No need to repeat the error with %v as it only can be the one that is already explain in the message.